packages/astro/src/content/runtime.ts

```diff
@@ -303,11 +301,9 @@ export function createGetEntry({
 				return;
 			}
 
-			if (entry.filePath) {
-				// @ts-expect-error	virtual module
-				const { default: imageAssetMap } = await import('astro:asset-imports');
-				entry.data = updateImageReferencesInData(entry.data, entry.filePath, imageAssetMap);
-			}
+			// @ts-expect-error	virtual module
+			const { default: imageAssetMap } = await import('astro:asset-imports');
```

Comment:
If you want you can type this in `dev-only.d.ts` that's at the root of the package

Reply:
Ah thanks. til

Reply:
I'm not sure why, but those types don't seem to be picked up in this file